plugins/analysis-phonetic/src/test/java/org/elasticsearch/index/analysis/AnalysisPhoneticFactoryTests.java

```diff
@@ -33,8 +33,8 @@ public AnalysisPhoneticFactoryTests() {
     @Override
     protected Map<String, Class<?>> getTokenFilters() {
         Map<String, Class<?>> filters = new HashMap<>(super.getTokenFilters());
-        filters.put("beidermorse", PhoneticTokenFilterFactory.class);
-        filters.put("doublemetaphone", PhoneticTokenFilterFactory.class);
+        filters.put("beider_morse", PhoneticTokenFilterFactory.class);
+        filters.put("double_metaphone", PhoneticTokenFilterFactory.class);
```

Comment:
Sorry, I was wrong with my previous comment that this needs changing. This will actually fail the test now. I think the test is based on class name lookup, and the "_" doesn't work with this.